components/core/src/clp/clp/FileCompressor.hpp

```diff
@@ -50,6 +50,9 @@ class FileCompressor {
     );
 
 private:
+    // Constants
+    static constexpr size_t m_utf_max_validation_len = 4;
```

Comment:
```suggestion
    static constexpr size_t cUtfMaxValidationLen = 4;
```
naming convention for constexpr data members